manifest/docker_schema1.go

```diff
@@ -228,10 +228,26 @@ func (m *Schema1) Inspect(_ func(types.BlobInfo) ([]byte, error)) (*types.ImageI
 		Architecture:  s1.Architecture,
 		Os:            s1.OS,
 		Layers:        layerInfosToStrings(m.LayerInfos()),
+		LayersDetail:  imgInspectLayersFromLayerInfos(m.LayerInfos()),
+		History:       nil,
```

Comment:
If `nil` is means “history is unavailable” and not “a zero-length history”, that needs to be documented in `types.go`.

Actually history exists in the image, compare `Schema1.ToSchema2Config`.